scripts/yocto-build-and-test.sh

```diff
@@ -647,23 +647,11 @@ build_and_test_client() {
         run_bitbake $images_to_build
 
         if ${BUILD_DOCKER_IMAGES:-false}; then
-            # Legacy note on release_tool --set-version-of
```

Comment:
I am wondering if this should not be a part of some general "Mender 3.7" cleanup and tracked somewhere?

Reply:
Yeah, you are right in that we could have had an explicit clean-up task. I think however that it is still related to "release improvements" so it is somewhat justified to do it here.

These kind of clean-ups are hard to plan because one only find them when working on something else...

Reply:
Great, in this case - conversation resolved :)